client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -431,41 +447,37 @@ function mediaButton( editor ) {
 		editor.nodeChanged();
 	}
 
-	function toggleSizingControls( event ) {
+	function toggleSizingControls( increase, event ) {
 		if ( ! event.element || 'IMG' !== event.element.nodeName ) {
 			return;
 		}
 
 		const parsed = MediaSerialization.deserialize( event.element );
 
-		// Disable sizing toggles if the image is transient
-		let isHidden = parsed.media.transient;
-
-		// Disable sizing toggles if the image is of an unknown size
-		if ( ! isHidden ) {
-			isHidden = ! parsed.appearance.size || ! includes( SIZE_ORDER, parsed.appearance.size );
-		}
+		// Hide sizing toggles if the image is transient
+		const isHidden = parsed.media.transient;
+		this.classes.toggle( 'hidden', isHidden );
```

Comment:
Seems to work fine in my testing, but one subtle change here is that we were previously always passing a boolean value to `toggle`, and now may pass `undefined` (since `transient` is not always a property on the media object). I'm guessing TinyMCE simply checks for general truthiness, but might be good to verify anyways.